src/ansys/geometry/core/designer/component.py

```diff
@@ -1876,3 +1879,34 @@ def build_parent_tree(comp: Component, parent_tree: str = "") -> str:
                 lines.extend([f"|{'-' * (indent - 1)}(comp) {comp.name}" for comp in comps])
 
         return lines if return_list else print("\n".join(lines))
+
+    @protect_grpc
+    @min_backend_version(26, 1, 0)
+    def import_named_selections(self) -> None:
+        """Import named selections of a component.
+
+        When a design is inserted, it becomes a component. In 26R1, the named selections of that
+        component will be imported by default. If a file is opened that contains a component that
+        did not have its named selections imported, this method can be used to import them.
+        """
```

Comment:
Add a warning with the minimum version
```suggestion
        did not have its named selections imported, this method can be used to import them.

        Warnings
        --------
        This method is only available starting on Ansys release 26R1.
        """
```